### PR TITLE
[ALLUXIO-2878] Change RmrCommandTest.java to RmrCommandIntegrationTest.java

### PR DESCRIPTION
**Pull request by dayhost**
https://alluxio.atlassian.net/browse/ALLUXIO-2878

Rename file tests/src/test/java/alluxio/shell/command/RmrCommandTest.java to tests/src/test/java/alluxio/shell/command/RmrCommandIntegrationTest.java

### CODE REVIEW

**Comment by dayhost** [comment]
alluxio-bot, check this please

**Comment by aaudiber** [comment]
@dayhost Thanks for the contribution, do you mind signing the CLA? You can fill it out here: http://cla.alluxio.org/

**Comment by dayhost** [comment]
Already sign the CLA, thanks

**Comment by aaudiber** [comment]
@dayhost Sorry, can you try again and make sure you submit the form? The records indicate that the CLA has been sent to your email, but no response has been received yet

**Comment by aaudiber** [comment]
alluxio-bot, check this please

**Comment by aaudiber** [comment]
alluxio-bot, check this please

File: tests/src/test/java/alluxio/shell/command/UnsetTtlCommandIntegrationTest.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 /**
  * Test for unsetTtl command.
  */
-public final class UnsetTtlTest extends AbstractAlluxioShellTest {
+public final class UnsetTtlCommandIntegrationTest extends AbstractAlluxioShellTest {
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/shell/command/UnsetTtlCommandIntegrationTest.java]: please revert the UnsetTtlTest change for this PR

Comment (dayhost) [inline tests/src/test/java/alluxio/shell/command/UnsetTtlCommandIntegrationTest.java]: Sorry, why I need to revert the change?

Comment (aaudiber) [inline tests/src/test/java/alluxio/shell/command/UnsetTtlCommandIntegrationTest.java]: According to the title of the pull request, the purpose is to change RmrCommandTest to RmrCommandIntegrationTest. This change is unrelated to that, so it should not be included in the pull request.

Comment (dayhost) [inline tests/src/test/java/alluxio/shell/command/UnsetTtlCommandIntegrationTest.java]: Could you give me some advice about how to modify it? Thanks.

Comment (aaudiber) [inline tests/src/test/java/alluxio/shell/command/UnsetTtlCommandIntegrationTest.java]: The root of the problem is that you're making a pull request from your `master` branch. Whenever you make a pull request in Github, you should create a new branch just for that pull request. To create a new `rmr` branch, you can do

```bash
# Nickname the Alluxio/alluxio repository to "upstream"
git remote add upstream git@github.com:Alluxio/alluxio.git
# Get the latest updates from Alluxio/alluxio
git fetch upstream
git checkout upstream/master
# Create a local branch named rmr
git checkout -b rmr
```

Once you've made your changes, you can push the branch to your Github fork (dayhost/alluxio) by running

```bash
git push origin rmr
```

Once you've pushed the branch to your fork, you can use the Github UI to create pull request for that branch.

Before doing all of this, make sure your email is set up correctly by running

```
git config --global user.email "dayhost@users.noreply.github.com"
```
